src/window.c

```diff
@@ -190,6 +191,58 @@ int S2D_Show(S2D_Window *window) {
             window->on_mouse(event);
           }
           break;
+        case SDL_FINGERMOTION:
+         {
+            if (window->on_touch) {
+           float fx,dx, fy, dy; 
+           fx = e.tfinger.x * window->width; // TODO: Viewport in scaling
+           fy = e.tfinger.y * window->height;
+           dx = e.tfinger.dx * window->width;
+           dy = e.tfinger.dy * window->height;
+           S2D_GetMouseOnViewport(window, (int)fx, (int)fy, &mx, &my);
```

Comment:
Ah, didn't realize that, thanks. Running the test card, my MacBook actually triggers touch events as well, I guess because the touchpad is technically "multi-touch." The x and y coords are off, so I'm going to investigate why that is.